conda-forge.yml

```
@@ -1 +1,3 @@
 conda_forge_output_validation: true
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
+test_on_native_only: true
```

could you also add osx (both flavors)? thanks!
```suggestion
test_on_native_only: true
build_platform:
   osx_arm64: osx_64
 test_on_native_only: true
```